R/run_stress_test.R

```diff
@@ -36,6 +36,58 @@ run_stress_test <- function(asset_type,
                             shock_year = 2030,
                             term = 2,
                             company_exclusion = TRUE) {
+  warn_if_asset_type_is_longer_than_1(asset_type)
+  matched_asset_type <- match.arg(asset_type)
+
+  # Allow testing anything before the long computation that follows
+  if (is_dev_env()) {
+    return(invisible(matched_asset_type))
+  }
+
+  # FIXME: Avoid clutter from printed "messages" that the user can't suppress
+  x <- capture.output(
```

Comment:
It'd be nice to hear why we have so much printed output and confirm it's okay to suppress.

Reply:
We have the printed output because Jacob and me need it as long as we are still actively developing the model. Just today it helped me find another bug and I will even have to add more clutter for the next days.
Next week I will make sure to discriminate between output that is necessary for the user and may not be suppressed and outputs that we need only for development purpose which I will surpress when run from user mode. Till then please do not supress output by default.